### PR TITLE
More Updates on Readme and package.json

### PR DESCRIPTION
**Pull request by Nicanor008**
## What does this PR do?
- Gives more description to the user through the readme and package-json file

## Description
- update on README file
- removal of duplicates keywords on the package-json file

## Issue
Closes [#26](https://github.com/kemboijs/kemboijs-cli/issues/26#issue-476643100)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,23 @@ Also, we will aim at building utility that can enhance faster development of upc
...
 NB: When installing make sure you have admin priviledges otherwise, you will have to use `sudo npm i -g kemboijs-cli`
 
-Also, after make sure to edit the database url in file `src/models/sequelizeinstance.js`.
-You can now call or edit the database models. The created model is User, under `src/models/user.js`
+## Edit Database details
+Navigate to `src/models/sequelizeinstance.js`, edit the database name, username or password. 
+
+You can also add these credentials to the `.env` file.
+```
+  export DATABASE_URL = "postgres://user:pass@example.com:5432/dbname"
```
Comment (ezrqnkemboi) [inline README.md]: You can add javascript for javascript codelines
```javascript
  export DATABASE_URL = "postgres://user:pass@example.com:5432/dbname"
```

File: README.md [hunk 2]
```diff
@@ -14,8 +14,23 @@ Also, we will aim at building utility that can enhance faster development of upc
...
+```
+
```
Comment (ezrqnkemboi) [inline README.md]: add javascript after ```
More information here https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

```
Code and Syntax Highlighting
Code blocks are part of the Markdown spec, but syntax highlighting isn't. However, many renderers -- like Github's and Markdown Here -- support syntax highlighting. Which languages are supported and how those language names should be written will vary from renderer to renderer. Markdown Here supports highlighting for dozens of languages (and not-really-languages, like diffs and HTTP headers); to see the complete list, and how to write the language names, see the highlight.js demo page.

Inline `code` has `back-ticks around` it.
Inline code has back-ticks around it.

Blocks of code are either fenced by lines with three back-ticks ```, or are indented with four spaces. I recommend only using the fenced code blocks -- they're easier and only they support syntax highlighting.

```javascript
var s = "JavaScript syntax highlighting";
alert(s);
```
 
```python
s = "Python syntax highlighting"
print s
```
 
```
No language indicated, so no syntax highlighting. 
But let's throw in a <b>tag</b>.
```
var s = "JavaScript syntax highlighting";
alert(s);
s = "Python syntax highlighting"
print s
No language indicated, so no syntax highlighting in Markdown Here (varies on Github). 
But let's throw in a <b>tag</b>.
```

File: README.md [hunk 3]
```diff
@@ -46,6 +61,21 @@ You can now call or edit the database models. The created model is User, under `
   <img src="https://contributors-img.firebaseapp.com/image?repo=kemboijs/kemboijs-cli" width="100"/>
 </a>
 
+# How To Contribute
```
Comment (ezrqnkemboi) [inline README.md]: Good work here.

File: README.md (hunk 2 above)
Comment (Nicanor008) [reply README.md]: wow just learned something awesome here on markdowns languages, thanks for the resource

File: README.md (hunk 1 above)
Comment (Nicanor008) [reply README.md]: This is implemented.

File: README.md (hunk 2 above)
Comment (ezrqnkemboi) [reply README.md]: Welcome anytime.

**Pull request merged**